subprojects/build-scan-performance/build-scan-performance.gradle.kts

```diff
@@ -16,28 +18,30 @@ import org.gradle.gradlebuild.unittestandcompile.ModuleType
  * limitations under the License.
  */
 plugins {
-    id 'gradlebuild.classycle'
+    id("gradlebuild.classycle")
 }
 
-apply from: 'templates.gradle'
+apply(from = "templates.gradle")
 
-tasks.named("performanceReport").configure {
+tasks.withType<PerformanceReport>().named("performanceReport").configure {
     resultStoreClass = "org.gradle.performance.results.BuildScanResultsStore"
 }
 
 dependencies {
     // so that all Gradle features are available
-    performanceTestRuntime allTestRuntimeDependencies
+    rootProject.configurations.testRuntime.allDependencies.forEach {
```

Comment:
:x: Remove duplication of `rootProject.configurations.testRuntime.allDependencies`, since it's always computed it could be an extension in `buildSrc`.